stand/i386/zfsboot/zfsboot.c

```diff
@@ -839,15 +848,16 @@ main(void)
 	reboot();
 
     /*
-     * Try to exec /boot/loader. If interrupted by a keypress,
-     * or in case of failure, try to load a kernel directly instead.
+     * Try to exec a loader or the kernel directly.
+     * If interrupted by a keypress, or in case of failure,
+     * drop the user to a boot2 prompt.
```

Comment:
Change the comment to match the new code.  I'd suggest something like, "Loop through the list of loaders/kernel" and merge that in with the language of the existing comment.

Reply:
Added commit to improve comments.